doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,328 @@
+    "description": "The AIO requirements to run this connector",
+    "properties": {
+      "aioMinVersion": {
+        "type": "string",
+        "description": "The minimal (inclusive) version that can run this connector."
+      },
+      "aioMaxVersion": {
+        "type": "string",
+        "description": "The maximal (inclusive) version that can run this connector."
+      }
+    }
+  },
+  "inboundEndpoints": {
+    "type": "array",
+    "description": "The array of inbound endpoints in a device that this connector will be deployed for.",
+    "items": [
+      {
+        "type": "object",
+        "properties": {
+          "endpointType": {
+            "type": "string",
+            "description": "The type of this inbound endpoint."
+          },
+          "configSchemas": {
+            "type": "object",
+            "description": "The inbound endpoint level JSON schemas for the various 'additional configuration' fields. A JSON schema must be provided for each 'additional configuration' only if that field is used by this connector.",
+            "properties": {
+              "additionalConfigurationSchema": {
+                "$ref": "#/definitions/schema"
+              }
+            }
+          },
+          "assets": {
+            "type": "array",
+            "description": "The array of assets that reference this inbound endpoint",
+            "items": [
+              {
+                "type": "object",
+                "properties": {
+                  "assetName": {
+                    "type": "string",
+                    "description": "The name of the asset"
+                  },
+                  "configSchemas": {
+                    "type": "object",
+                    "description": "The JSON schemas for the asset-level stringified JSON fields. A JSON schema must be provided for each field only if that field is used by this connector.",
+                    "properties": {
+                      "datasetConfigSchema": {
+                        "$ref": "#/definitions/schema"
+                      },
+                      "dataPointConfigSchema": {
+                        "$ref": "#/definitions/schema"
+                      },
+                      "eventConfigSchema": {
+                        "$ref": "#/definitions/schema"
+                      },
+                      "streamConfigSchema": {
+                        "$ref": "#/definitions/schema"
+                      },
+                      "managementGroupConfigSchema": {
+                        "$ref": "#/definitions/schema"
+                      },
+                      "processControlConfigSchema": {
+                        "$ref": "#/definitions/schema"
+                      }
+                    }
+                  }
+                }
+              }
+            ]
+          },
+          "version": {
+            "type": "string",
+            "description": "The version of this inbound endpoint."
+          }
+        },
```

Comment:
Current discussion goes towards removing this. We need to stay in sync with ConnectorTemplate.

Reply:
I am not aligned with this. I added a comment on the thread.